packages/app/src/client/services/AdminCustomizeContainer.js

```diff
@@ -14,9 +14,10 @@ const logger = loggerFactory('growi:services:AdminCustomizeContainer');
  */
 export default class AdminCustomizeContainer extends Container {
 
-  constructor() {
+  constructor(appContainer) {
```

Comment:
You aren't using this. Erase all diffs in this file

Reply:
`appContainer` has been removed from constructor in AdminCustomizeContainer because it no longer needed